tests/core/converters/test_select.cpp

```diff
@@ -82,5 +82,37 @@ TEST(Converters, ATenNarrowStartScalarConvertsCorrectly) {
 
     auto trt = trt_results[0].reshape(jit_results[0].sizes());
 
+    ASSERT_TRUE(trtorch::tests::util::almostEqual(jit_results[0], trt, 2e-6));
+}
+
+TEST(Converters, ATenSliceConvertsCorrectly) {

```

Comment:
Can we have tests for some of the more advanced slicing in python? like `[-1:]`?

Reply:
Sure. will add

Reply:
Added support and testcases to handle negative index cases.